integration-cli/docker_cli_daemon_test.go

```diff
@@ -2848,3 +2849,36 @@ func (s *DockerDaemonSuite) TestExecWithUserAfterLiveRestore(c *check.C) {
 	out, err = s.d.Cmd("stop", "top")
 	c.Assert(err, check.IsNil, check.Commentf("Output: %s", out))
 }
+
+func (s *DockerDaemonSuite) TestRemoveContainerAfterLiveRestore(c *check.C) {
+	testRequires(c, DaemonIsLinux, IsOverlay)
```

Comment:
Isn't `IsOverlay` redundant since you are just setting overlay below?
Maybe `overlayFSSuported` instead?